src/components/Passports/index.js

```diff
@@ -15,7 +15,22 @@ import {
   eraseStoredValue
 } from "../../services/localStorage";
 
-class Passports extends Component {
+export function getDefaultPassport(account, passports) {
+  const passportId = getStoredValue("currentPassport", account);
+  let passport;
+
+  if (passportId) {
+    passport = passports
+      ? passports.find(passport => passport.id === passportId)
+      : null;
+  }
+  if (!passport && passports && passports.length === 0) {
+    passport = passports[0]
```

Comment:
I belive, you wanted to assign passport to a first one, but if  ``passports.length === 0``  then ``passports[0] === undefined``

Reply:
Oh definitely yes, thanks!